wgpu-core/src/instance.rs

```diff
@@ -579,9 +583,18 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         &self,
         adapter_id: AdapterId,
         desc: &DeviceDescriptor,
+        unsafe_extensions: UnsafeExtensions,
         trace_path: Option<&std::path::Path>,
         id_in: Input<G, DeviceId>,
     ) -> DeviceId {
+        if desc.extensions.contains(Extensions::UNSAFE_EXTENSIONS) {
```

Comment:
this should be `intersects`

Reply:
I can never keep those straight for the life of me...

Reply:
I've hit that problem many times as well.. I wonder if it would be better called `contains_some`/`contains_all` instead of `intersects`/`contains`, but that train is off gone by now.